src/arm64/bitmanipulation.h

```diff
@@ -55,6 +55,7 @@ really_inline bool add_overflow(uint64_t value1, uint64_t value2, uint64_t *resu
 #endif
 }
 
+#if 0 // Currently unused
```

Comment:
Maybe we could just delete this code section. We can always retrieve it from previous releases.